.github/workflows/ci.yml

```diff
@@ -72,21 +72,6 @@ jobs:
       shell: bash
       run: make relay_tests RELAY_HOST=relay.walletconnect.com PROJECT_ID=${{ secrets.PROJECT_ID }}
 
-    # Smoke tests
-    - name: Run smoke tests
```

Comment:
why smoke-tests removed from ci action

Reply:
correct me if I am wrong but we don't run them in this workflow. And we have a run smoke tests action in run tests without building